commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/test/util/CommerceTierPriceEntryTestUtil.java

```diff
@@ -77,7 +76,8 @@ public static CommerceTierPriceEntry upsertCommerceTierPriceEntry(
 	}
 
 	private static long _getGroupId(
-			Long commercePriceEntryId, String priceEntryExternalReferenceCode)
+			long companyId, Long commercePriceEntryId,
```

Comment:
`Long commercePriceEntryId` should be a primitive `long`, or a NPE would be thrown at `if (commercePriceEntryId > 0)` if the value was `null`

Reply:
See https://github.com/liferay/com-liferay-commerce/pull/1414